script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -670,22 +761,28 @@ I can't say for certain since I can't see it, but I'm sure that those 'lines' ar
 [sha:0359112af9e1511c90a7dea7ca694996a00709fc]{
 -- Page 482, Offset 17770.
 -- ……確かにその通りだ。
-...That's exactly right.
+...She's spot on.
 }
 [sha:acd4f305d002f36c48431957d204c041ef020d9d]{
 -- Page 482, Offset 17771.
 -- 線だけのように視えて、目をこらすと点が視える事がある。事実、ヴローヴとの最後の瞬間、ヤツの体には大渦のような“点”があった。
-Focusing only on the lines, I can see something like a point if I look closely enough. In fact, during the last moment against Vlov, there was a 'point' like a giant whirlpool in his body.
+When I'm concentrating on trying to see the lines,
+ if I try hard enough I also start to see points as well.
+ In fact, during the final moment I spent face to face with Vlov,
+ I could see a giant vortex-like 'point' inside his body.
 }
 [sha:a386b9e3da53e05457290961d7bc34fed50493f9]{
 -- Page 483, Offset 17772.
 -- 振り返ってみれば、それはヴローヴの時だけじゃない。
-Looking back, that wasn't the only time I saw it.
+Now that I think back on it, that wasn't the only time I've seen them.
 }
 [sha:c8df1db77d7f7b2f12c0261c9b25433144fdb902]{
 -- Page 483, Offset 17773.
 -- たいてい脳幹をぶった斬るような<幻痛|ショック>から意識を失うけど、点は確かにあった。
-Although most of the time I would lose consciousness from the shock that felt like my brain stem was being severed, I could definitely see the point.
+// Brainstem split up for more plantlike hacking metaphor
+Whenever I do, I'm assailed by a <phantom pain|s h o c k   >,
+ as though my brain is being hacked off at the stem, and lose consciousness.
```

Comment:
"and **I** lose consciousness" Without it reads like his _brain_ is the one losing consciousness which would require a tense change